index.html

```diff
@@ -1,41 +1,79 @@
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="shortcut icon" href="img/FoodIco.png" type="image/x-icon">
-  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
-  <link rel="stylesheet" href="css/jquery-ui.min.css">
-  <link rel="stylesheet" href="css/styles.css">
-  <script src="https://kit.fontawesome.com/3894c1dcb1.js" crossorigin="anonymous"></script>
-  <title>Interactive frontend application</title>
-</head>
-<body>
-  <h1>Interactive frontend application (name pending...)</h1>
-
-  <main>
-    <!-- !!! remove before writing actual code !!! -->
-    <div class="card" style="width: 18rem;">
-      <div class="card-body">
-        <h5 class="card-title">Lorem Ipsum</h5>
-        <h6 class="card-subtitle mb-2 text-muted">Lorem Ipsum</h6>
-        <p class="card-text">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Necessitatibus repellat, voluptate minima molestiae odit alias iste mollitia itaque, quisquam porro placeat.</p>
-        <a href="#" class="card-link">Lorem Ipsum</a>
-        <a href="#" class="card-link">Lorem Ipsum</a>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="shortcut icon" href="img/FoodIco.png" type="image/x-icon" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="css/jquery-ui.min.css" />
+    <link rel="stylesheet" href="css/styles.css" />
+    <link rel="stylesheet" href="css/mainPage.css" />
+    <script
+      src="https://kit.fontawesome.com/3894c1dcb1.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Interactive frontend application</title>
+  </head>
+  <body>
+
+    <main class="container-fluid">
+      <div class="container" id="homeScreen">
+        <div class="row logoContainer">
+          <div class="col-3 align-self-start">This will be the Logo Div InFridge</div>
+        </div>
+        <div class="row justify-content-center">
+          <div class="col-8 descriptionText">
+            Type your Ingredient in the Box below and hit Search for inspiration!
+          </div>
+          <form class="row search justify-content-center"> 
+            <div class="col-6">
+              <input type="text" class="form-control searchInput" id="searchValue" placeholder="Search Ingredient">
+            </div>
+            <div class="col-2 d-grid">
+              <button type="button" class="btn btn-primary btn-lg btn-block searchBtn searchButton">Search!</button>

```

Comment:
You are calling a class of searchButton, but you never defined it in css.

Reply:
amended